src/Data/Bin/Properties.agda

```diff
@@ -602,7 +597,8 @@ fromℕ-homo-* m n = begin
 *-distrib-+ = *-distribˡ-+ , *-distribʳ-+
 
 ------------------------------------------------------------------------
--- Structures
+-- Instances for Bin for some classical algebraic categories.
+-- Multiplicative part, CommutativeSemiring.
```

Comment:
We've had this discussion twice before now. Please could you keep the standard section headers.

Reply:
I copy your headers now.
By the way, which of these headers in your file are standard?
```
------------------------------------------------------------------------
-- Properties of _≡_
------------------------------------------------------------------------
```
,
```
------------------------------------------------------------------------
-- Properties of _+_
------------------------------------------------------------------------

------------------------------------------------------------------------
-- toℕ/fromℕ are homomorphisms for _+_
```
,
```
------------------------------------------------------------------------
-- Algebraic properties of _+_

-- Mostly proved by using the isomorphism between `ℕ` and `Bin` provided
-- by `toℕ`/`fromℕ`.
```
,

```
------------------------------------------------------------------------
-- Structures
```

Reply:
The standard ones are
```
------------------------------------------------------------------------
-- Properties of _≡_
------------------------------------------------------------------------

------------------------------------------------------------------------
-- Properties of _+_
------------------------------------------------------------------------

------------------------------------------------------------------------
-- Algebraic properties of _+_

------------------------------------------------------------------------
-- Structures
```

Reply:
I hope, now it agrees with the style.